src/components/veiledning/ScanningIcon.tsx

```diff
@@ -0,0 +1,151 @@
+                    <feGaussianBlur stdDeviation={2} in="shadowOffsetOuter1" result="shadowBlurOuter1" />
+                    <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0" in="shadowBlurOuter1" />
+                </filter>
+                <path id="keystone_2__b" d="M16.528 20h49.06L78 89H6z" />
+            </defs>
+            <g fill="none" fillRule="evenodd">
+                <path d="M1 1h81v119H1z" fill="#BABABA" stroke="#000" strokeOpacity={0.85} strokeWidth={2} />
+                <use fill="#000" filter="url(#keystone_2__a)" xlinkHref="#keystone_2__b" />
+                <use fill="#FFF" xlinkHref="#keystone_2__b" />
+                <g stroke="#B7B4B4" strokeLinecap="square" strokeWidth={2}>
+                    <path d="M20 41.5h37M21 35.5h39M22 29.5h36M19 47.5h43" />
+                </g>
+                <g stroke="#B7B4B4" strokeLinecap="square" strokeWidth={2}>
+                    <path d="M15 70.5h48M16 64.5h45M17 58.5h38M14 76.5h45" />
+                </g>
+            </g>
+        </svg>
+    );
+};
+const HorizontalScanning = ({ title, height }: ScanProps) => {
+    return (
+        <svg role="presentation" aria-hidden focusable="false" viewBox="0 0 83 121" height={height}>
+            {title && <title>{title}</title>}
+            <desc>Bildet har ikke riktig retning</desc>
+            <defs>
+                <filter
+                    x="-10.6%"
+                    y="-11.1%"
+                    width="121.2%"
+                    height="131.1%"
+                    filterUnits="objectBoundingBox"
+                    id="horizontal_3__a"
+                >
+                    <feOffset dy={2} in="SourceAlpha" result="shadowOffsetOuter1" />
+                    <feGaussianBlur stdDeviation={2} in="shadowOffsetOuter1" result="shadowBlurOuter1" />
+                    <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0" in="shadowBlurOuter1" />
+                </filter>
+                <path id="horizontal_3__b" d="M8 36h66v45H8z" />
+            </defs>
+            <g fill="none" fillRule="evenodd">
+                <path d="M1 1h81v119H1z" fill="#BABABA" stroke="#000" strokeOpacity={0.85} strokeWidth={2} />
+                <use fill="#000" filter="url(#horizontal_3__a)" xlinkHref="#horizontal_3__b" />
+                <use fill="#FFF" xlinkHref="#horizontal_3__b" />
+                <path
+                    stroke="#B7B4B4"
+                    strokeWidth={2}
+                    strokeLinecap="square"
+                    d="M35 44v28M30 44v22M25 44v19M20 44v24M52 44v24M57 44v30M47 44v27M62 44v26"
+                />
+            </g>
+        </svg>
+    );
+};
+const ShadowScanning = ({ title, height }: ScanProps) => {
+    return (
+        <svg role="presentation" aria-hidden focusable="false" viewBox="0 0 83 121" height={height}>
+            {title && <title>{title}</title>}
+            <desc>Bildet har har skygge oppå legeerklæring</desc>
```

Comment:
Vet ikke hva desc brukes til av nettleseren, men legeerklæring er vel neppe relevant for oss (tipper koden her stammer fra pleiepenger?).

Reply:
Fiksa desc